Classes/Task/DeutscherWetterdienstTaskAdditionalFieldProvider.php

```diff
@@ -128,14 +128,13 @@ public function getAdditionalFields(
 
     protected function initialize(): void
     {
-        $this->pageRenderer->loadRequireJs();
         $this->pageRenderer->addInlineLanguageLabelFile(
             'EXT:weather2/Resources/Private/Language/locallang_scheduler_javascript_deutscherwetterdienst.xlf',
         );
         $this->pageRenderer->addCssFile('EXT:weather2/Resources/Public/Css/dwdScheduler.css');
-        $this->pageRenderer->loadRequireJsModule('TYPO3/CMS/Backend/FormEngineValidation');
-        $this->pageRenderer->loadRequireJsModule('TYPO3/CMS/Weather2/jquery.autocomplete');
-        $this->pageRenderer->loadRequireJsModule('TYPO3/CMS/Weather2/DeutscherWetterdienstTaskModule');
+        $this->pageRenderer->loadJavaScriptModule('TYPO3/CMS/Backend/FormEngineValidation');
+        $this->pageRenderer->loadJavaScriptModule('TYPO3/CMS/Weather2/jquery.autocomplete');
```

Comment:
I thought with TYPO3 13 jquery is available, but not loaded by default anymore. Can you confirm that this JS is still working? Further I don't see a `JavaScriptModules.php` file allowing to load these JS files.